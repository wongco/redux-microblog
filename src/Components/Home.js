import React, { Component } from 'react';
import TitleList from '../Containers/TitleList';
import styled from 'styled-components';

const StyledHome = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 100%;
`;

class Home extends Component {
  render() {
    return (
      <StyledHome>
        <p>
          Welcome to <b>Microblog</b>, our innovative site for communicating on
          the information superhighway.
        </p>
        <TitleList />
      </StyledHome>
    );
  }
}

export default Home;
