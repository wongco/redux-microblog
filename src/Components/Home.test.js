import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from './Home';

it('renders without crashing', () => {
  shallow(<Home />);
});

it('matches snapshot for default criteria', () => {
  const wrapper = shallow(<Home />);
  const serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
