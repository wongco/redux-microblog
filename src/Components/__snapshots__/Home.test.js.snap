// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for default criteria 1`] = `
<styled.div>
  <styled.p>
    Welcome to 
    <b>
      Microblog
    </b>
    , our innovative site for communicating on the information superhighway.
  </styled.p>
  <Connect(TitleList) />
</styled.div>
`;
